tests/cucumber/features/authentication/authentication.feature

```diff
@@ -7,3 +6,13 @@ Feature: Login
   Background:
     Given I am on the home page
     Given I am signed out
+
+  @dev
+  Scenario: Create account
+    Given "test@test.com" has not registered
+    When I click on "a.dropdown-toggle"
+    And I click on "#signup-link"
+    And I enter "test@test.com" in "#login-email"
+    And I enter "testpass" in "#login-password"
+    And I click on "button#login-buttons-password"
+    # Then The element "a.dropdown-toggle" should have text "test@test.com"
```

Comment:
This element keeps coming back as 'Sign in / Join'.

After doing a little digging, there is an error code saying that the email is already in use.